src/System.Xml.XmlSerializer/src/System/Xml/Serialization/XmlAnyElementAttributes.cs

```diff
@@ -16,7 +17,7 @@ namespace System.Xml.Serialization
     /// </devdoc>
     public class XmlAnyElementAttributes : IList
     {
-        private ArrayList _list;
+        private List<object> _list;
```

Comment:
> He just changed MemberMappingComparer to implement the generic interface here; as long as it isn't exposed in the contract (which it isn't), it shouldn't matter, right?

Technically, it shouldn't matter, but personally, I wouldn't implement the generic `IList<T>` until it's in the contract. `MemberMappingComparer` is different because it's internal and it changed for a reason: to be used with the generic `Sort` method.

> Also the IList-based methods that take an object don't appear to be exposed, either.

Ah, I didn't realize these collections don't implement the non-generic `IList` in the contract. In that case, it could use the more strict `List<T>` typing. And the `IList` implementation could be removed as it's just dead code in .NET Core (probably as a separate PR).